File: src/main/java/com/procurement/contracting/model/dto/CreateCanRS.java
package com.procurement.contracting.model.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.List;
import lombok.Getter;

@Getter
public class CreateCanRS {

    @JsonProperty("cans")
    private final List<Can> cans;

    public CreateCanRS(@JsonProperty("cans") final List<Can> cans) {
        this.cans = cans;
    }
}

File: src/main/kotlin/com/procurement/contracting/model/dto/CreateCanRS.kt
package com.procurement.contracting.model.dto

import com.fasterxml.jackson.annotation.JsonInclude
import com.procurement.contracting.model.dto.ocds.Can

@JsonInclude(JsonInclude.Include.NON_NULL)
data class CreateCanRS(

        val cans: List<Can>?
)
